packages/react-native-bpk-component-calendar/stories.js

```diff
@@ -270,6 +270,18 @@ storiesOf('react-native-bpk-component-calendar', module)
       ]}
     />
   ))
+  .add('With selected dates (UTC number)', () => (
+    <BpkCalendarExample
+      style={styles.calendarOnly}
+      selectionType={SELECTION_TYPES.range}
+      minDate={new Date(Date.UTC(2019, 0, 2))}
```

Comment:
This seems to be inconsistent with the test description? `new Date(Date.UTC(2019, 0, 2))` is a `Date` not a `number`. Shouldn't this be just `minDate={Date.UTC(2019, 0, 2)}`?

Reply:
Good point